plugins/ltac/g_obligations.mlg

```diff
@@ -77,14 +76,10 @@ let classify_obbl _ = Vernacextend.(VtStartProof (Doesn'tGuaranteeOpacity,[]))
 }
 
 VERNAC COMMAND EXTEND Obligations CLASSIFIED BY { classify_obbl } STATE declare_program
-| [ "Obligation" natural(num) "of" identref(name) ":" lglob(t) withtac(tac) ] ->
-    { obligation (num, Some name.CAst.v, Some t) tac }
 | [ "Obligation" natural(num) "of" identref(name) withtac(tac) ] ->
```

Comment:
this grammar rule has not changed and gives access to `with tac`

The doc on master makes it look like the type is need to use `with tac` but that seems to be a doc-only bug

Reply:
Ah, good! Then OK to merge this without needing a deprecation phase.